### DISCUSSION TITLE
Select Video Quality on HLS/Dash

### DISCUSSION THREAD

**Discussion by zaied45-183** [yoriiis/vlitejs#111] [Ideas]
This is a must-have feature for HLS players. Does Vlite have any option to switch among different quality levels? As well as when we select auto it should return on Adaptive mode.

![image](https://github.com/vlitejs/vlite/assets/59955142/bde998c0-2f20-45cb-abaa-6228b7e0200f)

**Comment by saierirfan00** [comment]
Isn't it available at the moment? I'm trying to enable quality switcher? Did you find out any solution?

**Comment by yoriiis** [comment]
> This is a must-have feature for HLS players. Does Vlite have any option to switch among different quality levels? As well as when we select auto it should return on Adaptive mode.

It could be a plugin, see [create a custom plugin](https://github.com/vlitejs/vlite/blob/main/src/plugins/README.md#create-a-custom-plugin)
